### PR TITLE
Remove unused old code from gravity.sh

### PR DESCRIPTION
**Pull request by rdwebdesign**
### What does this PR aim to accomplish?

Remove unused/dead code from `gravity.sh`, forgot after a modification, 3 years ago.

Context:
The first `if` block inside the function was commented, making the code unconditional.
After that edition, all code after the `return 0` was never executed and became useless (but we kept updating it!)

### How does this PR accomplish the above?

Removing the unused code.

### What documentation changes (if any) are needed to support this PR?

none

---
**By submitting this pull request, I confirm the following:** 

1. I have read and understood the [contributors guide](https://docs.pi-hole.net/guides/github/contributing/), as well as this entire template. I understand which branch to base my commits and Pull Requests against. 
2. I have commented my proposed changes within the code and I have tested my changes.
3. I am willing to help maintain this change if there are issues with it later.
4. It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
5. I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

---
- [x] I have read the above and my PR is ready for review. _Check this box to confirm_

### CODE REVIEW

**Comment by yubiuser** [comment]
Relevant PR: https://github.com/pi-hole/pi-hole/pull/2935

File: gravity.sh [hunk 1]
```diff
@@ -720,72 +720,26 @@ gravity_DownloadBlocklistFromUrl() {
...
+  # 5) Remove leading tabs, spaces, etc.
+  # 6) Delete lines not matching domain names
+  < "${src}" tr -d '\r' | \
+  tr '[:upper:]' '[:lower:]' | \
+  sed 's/\s*#.*//g' | \
+  sed -r '/(\/).*$/d' | \
+  sed -r 's/^.*\s+//g' | \
+  sed -r '/([^\.]+\.)+[^\.]{2,}/!d' >  "${destination}"
+  chmod 644 "${destination}"
+  return 0
```
Comment (yubiuser) [inline gravity.sh:742]: Do we still need `return 0` here?
```suggestion
  
```

Comment (rdwebdesign) [reply gravity.sh:742]: Good question and the answer is: no.
This return value is never used... I'll remove it.

**Review by yubiuser — APPROVED** [review]

**Review by DL6ER — APPROVED** [review]
As being guilty for not removing it three years ago, I hereby approve.

**Pull request merged**